Podfile.lock

```diff
@@ -51,4 +51,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 49f35055dbccfeeb8fddde878795bc7af6351950
 
-COCOAPODS: 1.2.1
+COCOAPODS: 1.3.1
```

Comment:
In case if we don't decide to upgrade, it might be worth downgrading and reinstalling the pods, as version mismatch might cause issues with CI.

Reply:
FWIW this is controlled by the circle.yml file. Without going into detail (because I honestly don't remember either, I'd look it up on Circle's site), this needs to be updated for CI to run `pod install` with 1.3.1. we voted to get that done, so it's on my list, but I haven't gotten to yet. just FYI from my end on this :)